exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,37 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if(number<0){
+return true;
```

Comment:
careful with your formatting
VS code has formatting built in and you can also get extensions
helps readability